operations.go

```diff
@@ -41,6 +41,18 @@ type (
 	opmapType map[string]ophandler
 )
 
+func fib(x uint64) uint64 {
+	if x < 1 || x > 45 {
+		// less than one is outside the domain of the mathematical function,
+		// and more than 45 overflows uint64 and results in infinite recursion
+		panic("fib() called with too small or large value!")
```

Comment:
Panic will crash the program with a huge and ugly stack trace. It's never the best solution. If we want to quit the program with a fatal condition, log.Fatal is the way to go. Here, I think this should return an error that is then "bubbled up" to the caller and displayed to the user.